.github/workflows/release-chart.yaml

```
@@ -7,6 +7,9 @@ on:
     paths:
       - charts/metrics-server/Chart.yaml
 
+permissions:
+  contents: write
```

```suggestion
permissions:
  contents: read
```

The GitHub best practice is for following block to be added to the job while the workflow get least privilege.

```yaml
permissions:
  contents: write
```

---

I see, I thought it was only a top-level thing